src/coverlet.msbuild.tasks/CoverageResultTask.cs

```diff
@@ -7,13 +7,15 @@
 using Coverlet.Core.Reporters;
 using Microsoft.Build.Framework;
 using Microsoft.Build.Utilities;
+using Palmmedia.ReportGenerator.Core;
```

Comment:
I don't think coverlet should have a dependency on this specific report generator.